src/services/measures/attitude/client/attitude.js

```diff
@@ -1,25 +1,85 @@
-tash.namespace('Gauges.Widgets');
+tash.namespace("Gauges.Widgets");
 
```

Comment:
Oki, no problem mate. thanks :D  
PS: out of curiosity, the most classic question of all: "what code editor are u using ?" :)